samza-core/src/main/java/org/apache/samza/config/TaskConfigJava.java

```diff
@@ -37,7 +37,7 @@
 public class TaskConfigJava extends MapConfig {
   // Task Configs
   private static final String TASK_SHUTDOWN_MS = "task.shutdown.ms";
-  public static final long DEFAULT_TASK_SHUTDOWN_MS = 5000L;
+  public static final long DEFAULT_TASK_SHUTDOWN_MS = 30000L;
```

Comment:
@shanthoosh  (just re-iterating our offline conversation) Imo, `task.shutdown.ms` should not be affected by executing environment. Changing defaults across environment in the code will confuse the user, even if it doesn't directly affect samza-yarn jobs. So, if we want to make the defaults reasonable to avoid unnecessary re-balancing attempts in standalone, it should be fine as long as it is called out clearly everywhere. 

Again, this may still impact large through-put job which can easily start lagging. This is purely due to our architectural changes. In case of Yarn, there is no question of "rebalancing" today. So, we have to find a balance that will work for both architectural choices.